specification/columns/commitmentutilizationused.md

```diff
@@ -0,0 +1,48 @@
+# Commitment Utilization Used
+
+Commitment Utilization Used is a numeric value representing the unit amount of a purchased [*commitment-based discount*](#glossary:commitment-based-discount) applied or not applied to a resource during a billing period.  The ratio of CommitmentUtilizationUsed over [*CommitmentUtilizationTotal*](#commitment-utilization-total) produces the utilization rate for a given commitment-based discount or aggregate of.
+
+CommitmentUtilizationTotal column MUST be present in the billing data when the provider supports *commitment-based discounts*.
+
+A positive CommitmentUtilizationUsed value MUST be applied to a [*row*](#glossary:row) when:
+
+* ChargeCategory is *Usage*
+* ChargeSubCategory is *UsedCommitment* or *UnusedCommitment*.
```

Comment:
Do you want unused commitment to contribute to this column?

Reply:
Removing